pymc3/step_methods/nuts.py

```diff
@@ -0,0 +1,238 @@
+from .quadpotential import quad_potential
```

Comment:
I bet you accidentally checked in this file with a `git commit -a`, at least that happens to me all the time.